versions/1/changelog.yml

```diff
@@ -2,6 +2,11 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom.
 ##
+- version: 1.8.0
+  changes:
+  - description: Add Kibana/osquery-saved-query
```

Comment:
Let's add this entry to the 1.7.1-next. Eventually, it will become 1.8.0.

Reply:
Ok!